statsmodels/tsa/stattools.py

```diff
@@ -1066,12 +1174,12 @@ def arma_order_select_ic(y, max_ar=4, max_ma=2, ic='bic', trend='c',
     adftstat = adfuller(x, autolag="t-stat")
 
 # acf is tested now
-    acf1, ci1, Q, pvalue = acf(x, nlags=40, confint=95, qstat=True)
-    acf2, ci2, Q2, pvalue2 = acf(x, nlags=40, confint=95, fft=True, qstat=True)
-    acf3, ci3, Q3, pvalue3 = acf(x, nlags=40, confint=95, qstat=True,
-                                 unbiased=True)
-    acf4, ci4, Q4, pvalue4 = acf(x, nlags=40, confint=95, fft=True, qstat=True,
-                                 unbiased=True)
+#    acf1, ci1, Q, pvalue = acf(x, nlags=40, confint=95, qstat=True)
+#    acf2, ci2, Q2, pvalue2 = acf(x, nlags=40, confint=95, fft=True, qstat=True)
+#    acf3, ci3, Q3, pvalue3 = acf(x, nlags=40, confint=95, qstat=True,
+#                                 unbiased=True)
+#    acf4, ci4, Q4, pvalue4 = acf(x, nlags=40, confint=95, fft=True, qstat=True,
+#                                 unbiased=True)
 
```

Comment:
The confint argument to `act()` is no longer used. It was in a MAINT note, so I figured I'd just update it by commenting them out.